docs/source/install/profile.rst

```diff
@@ -69,7 +69,7 @@ For this profile to work, you need to download the :ref:`PIConGPU source code <i
 
 .. note::
 
-   Please find a `Piz Daint quick start from December 2017 here <https://gist.github.com/ax3l/68cb4caa597df3def9b01640959ea56b>`_.
+   Please find a `Piz Daint quick start from December 2017 here <https://gist.github.com/n01r/20dacc20987b75d1858db5261d5d0205>`_.
```

Comment:
@n01r / @steindev can you please revert this line but keep the change below?

Reply:
@n01r merged :)
https://gist.github.com/ax3l/68cb4caa597df3def9b01640959ea56b
(gists are git repos, did a command-line `git merge` of your branch, solved conflicts and added some clean-ups :-) )

Reply:
Done